### PR TITLE
utils_test.qemu: add session for setup_win_driver_verifier

### PR DESCRIPTION
**Pull request by kimi1978wy**
add session for setup_win_driver_verifier and encapsulate it.

Signed-off-by: Yu Wang <wyu@redhat.com>

id:1557143

### CODE REVIEW

**Comment by kimi1978wy** [comment]
related cases results:
(1/1) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.Win2008.i386.sp2.io-github-autotest-qemu.win_irq_check.with_balloon: PASS (434.44 s)
(1/1) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.Win2008.i386.sp2.io-github-autotest-qemu.balloon_service.sc_interrogate: PASS (636.01 s)
(1/1) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.Win2008.i386.sp2.io-github-autotest-qemu.pvpanic_basic.nmi: PASS (629.14 s)

more results refer to https://github.com/autotest/tp-qemu/pull/1257

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

File: virttest/utils_test/qemu/__init__.py [hunk 1]
```diff
@@ -137,33 +137,30 @@ def _check_driver_verifier(session, driver, timeout=300):
...
-                                                                   output)
-                raise exceptions.TestFail(msg)
-        logging.info("%s verifier is enabled already" % driver)
-    finally:
-        session.close()
+            msg = "%s verifier is not enabled, details: %s" % (driver,
+                                                               output)
+            raise exceptions.TestFail(msg)
+    logging.info("%s verifier is enabled already" % driver)
+    return session
```
Comment (xiagao) [inline virttest/utils_test/qemu/__init__.py]: hi, why return session?

Comment (kimi1978wy) [reply virttest/utils_test/qemu/__init__.py]: if guest reboot after driver verifier, should return session, since the session changed.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**